server/lib/ga-collector.js

```diff
@@ -80,6 +81,8 @@ GACollector.prototype = {
           uid: body.uniqueUserId
         };
 
+        gaData.qt = self._calculateQueueTime(body.startTime, event.offset);
```

Comment:
Huh, interesting. We convert navigation timing data from wall time to offset, back to wall time.